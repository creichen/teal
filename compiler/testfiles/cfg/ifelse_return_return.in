fun f(x) = {
  if (x) {
    return 0;
  } else {
    return 1;
  }
}

