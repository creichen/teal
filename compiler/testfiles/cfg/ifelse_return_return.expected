Entry#0 @ 1 :
  -> Access#0
VarDecl#0 @ 1 :
Access#0 @ 2 :
  -> IntConstant#0
  -> IntConstant#1
ReturnStmt#0 @ 3 :
  -> Exit#0
IntConstant#0 @ 3 :
  -> ReturnStmt#0
ReturnStmt#1 @ 5 :
  -> Exit#0
IntConstant#1 @ 5 :
  -> ReturnStmt#1
Exit#0 @ 7 :
